internal/cmd/api-key/command_delete.go

```diff
@@ -39,9 +40,12 @@ func (c *command) delete(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	deleted, err := resource.Delete(args, deleteFunc, c.postProcess)
-	resource.PrintDeleteSuccessMsg(deleted, resource.ApiKey)
+	deletedIDs, err := resource.Delete(args, deleteFunc)
```

Comment:
```suggestion
	deletedIds, err := resource.Delete(args, deleteFunc)
```

Reply:
It feels weird to be waiting to handle this error.

Reply:
You left feedback last time (that I accidentally marked resolved) about rolling the error message into the `Delete` function (and I think Muwei asked for that too).

After I do that, this error will be handled immediately in most calls. The only ones where it won't be will be for resources like api-key where I need to do further tasks, like deleting the keys from the key store.

Since some deletes can succeed and some can fail, I can't return the error until I've removed the correct keys from the key store and added any potential new errors to the multierror.